docs/autojustice.rst

```diff
@@ -1,22 +1,50 @@
 autojustice
-========
+===========
 
 .. dfhack-tool::
     :summary: Auto-manage justice interviews and convictions.
     :tags: fort auto
 
-This script automate most of the tedious parts of the justice system. It schedules 
-visitors interviews, convict confessed crimes and interview accused units.
+This script automate most of the tedious parts of the justice system. It schedules visitors interviews, convict confessed crimes and interview accused units.
 
 Usage
 -----
 ``enable autojustice``
     Enable the script
 ``disable autojustice``
     Disable the script
+``autojustice [<options>]``
+    Set conviction options
+
+Options
+-------
+
+``-j``, ``--jailcitizen``
```

Comment:
in the code, you have these options taking an argument, but I don't see it documented here. This should be:
```
``-j``, ``--jailcitizen true|false``
```
now that you have configuration options, you should also write a simple GUI to configure them, which should also show statistics and other useful information. See `gui/autofish` for an example. That can come in a different PR, though, if you like

Reply:
I will take a look on the GUI part later on. 

Should I still add the "autojustice" entry to the `gui/controrl-panel` even without a GUI?

Reply:
Yes, the control panel will automatically pick up the GUI config when it's added. Until then, autojustice will just not have a button for launching the gui